pkg/kube/controllers/boshdeployment/bpm_controller.go

```diff
@@ -55,7 +56,13 @@ func AddBPM(ctx context.Context, config *config.Config, mgr manager.Manager) err
 		CreateFunc: func(e event.CreateEvent) bool {
 			o := e.Object.(*corev1.Secret)
 			shouldProcessEvent := isBPMInfoSecret(o)
+
 			if shouldProcessEvent {
+				annotations := o.GetAnnotations()
```

Comment:
This will skip all resources, which have been previously reconciled? Interesting.

Instead of GetAnnotations.. I think you could use `metav1.HasAnnotation` from 
`k8s.io/apimachinery/pkg/apis/meta/v1/helpers.go`.

Reply:
Yeah. we do not want reconcile these create events again when operator restarts. Done.